client/src/components/Profile/Calender/Calender.tsx

```diff
@@ -23,7 +23,7 @@ export const Calender = () => {
     title: '',
     description: '',
   });
-  const [open, setOpen] = useState<IOpen>({
+  const [open, setOpenPop] = useState<IOpen>({
```

Comment:
rename the `open` to `openPop`

Reply:
or destructure it

Reply:
i think this is my code :+1: 
can i take this ?